pkg/apis/meta/v1alpha1/image.go

```diff
@@ -47,3 +45,89 @@ func (i *OCIImageRef) UnmarshalJSON(b []byte) error {
 	*i, err = NewOCIImageRef(str)
 	return err
 }
+
+func ParseOCIContentID(str string) (*OCIContentID, error) {
```

Comment:
add a short godoc showing sample valid/invalid strings you can give it?

Reply:
Done in d4495d0.